styles/custom-viewer-styles.css

```
@@ -1,7 +1,33 @@
 .viewer-container img {
-  -webkit-user-drag: none;
-  user-select: none;
-  -moz-user-select: none;
+  /* Prevent long-press context menu on mobile */
+  -webkit-touch-callout: none;
   -webkit-user-select: none;
+  -khtml-user-select: none;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)